cdci_data_analysis/analysis/parameters.py

```diff
@@ -674,29 +664,24 @@ def _set_angle(self, value, units):
 #
 
 
+class SpectralBoundary(Float):
+    def __init__(self, value=None, E_units='keV' ,name=None):
 
-class SpectralBoundary(Parameter):
-    def __init__(self,value=None,E_units='keV',name=None):
-
-        _allowed_units = ['keV','eV','MeV','GeV','TeV','Hz','MHz','GHz']
+        _allowed_units = ['keV', 'eV', 'MeV', 'GeV', 'TeV', 'Hz', 'MHz', 'GHz']
```

Comment:
there is some confusion between `type` and `units`. `SpectralBoundary` has type `float` but units e.g. `keV`.

Reply:
so perhaps we should rename it `default_format` from `default_units`?

Reply:
yeah. Well I'd call it `type`: it's not exactly the same as format.
As it's inherited from `Float`, float should take care of the entire business of having parameter type. Float, in principle, may have no unit. But anything with unit would also be a float.

Reply:
Same, as for the `Float` Parameter, applies to `Integer`.

Reply:
> so perhaps we should rename it `default_format` from `default_units`?
>
Even better, having them both allows to have a unit as well as being a float (or integer).

Reply:
both `default_type` and `default_units` ? yes.